src/main/java/org/prebid/server/bidder/invibes/InvibesBidder.java

```diff
@@ -121,6 +121,15 @@ private void validateImp(Imp imp) {
         }
     }
 
+    private boolean isAmp(BidRequest request) {
+        if (request.getExt() == null) {
+            return false;
+        } else if (request.getExt().getPrebid() == null) {
+            return false;
+        }
+        return request.getExt().getPrebid().getAmp() != null;
```

Comment:
Not our way of doing such things, we prefer solutions like in `resolveConsentString` or wrapping with `Optional` and then chaining `.map` calls.

Reply:
Oh nice :tada: ! Last time I used `Optional` it was marked in the code review.

Reply:
Yeah, we decided to start using `Optional` recently.